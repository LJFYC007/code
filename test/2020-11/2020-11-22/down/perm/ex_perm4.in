3
11
11 5 1 4 3 9 2 6 7 10 8
1 11 5 3 4 7 8 2 9 6 10
45
20 3 17 23 21 1 7 35 34 26 27 38 15 4 13 11 16 14 33 2 44 30 28 25 6 45 37 29 40 12 9 31 19 42 36 8 5 22 41 39 18 43 10 32 24
4 39 13 6 31 32 20 45 22 1 42 11 12 17 14 10 9 15 5 24 43 38 35 18 19 33 28 36 7 8 29 41 40 26 25 23 3 30 44 21 27 2 37 34 16
14
2 14 12 8 6 3 13 5 7 11 9 10 1 4
3 9 1 4 2 5 8 11 6 13 10 12 14 7
