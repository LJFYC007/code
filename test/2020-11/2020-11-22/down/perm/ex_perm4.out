4891912
869271467
927434396
